dimod/binary/binary_quadratic_model.py

```diff
@@ -860,50 +879,45 @@ def add_variable(self, v: Optional[Variable] = None, bias: Bias = 0):
         """Add a variable to a binary quadratic model.
 
         Args:
-            v: Variable label. If not provided, the next interger label
+            v: Variable label. If not provided, the next integer label
                 is used.
-
             bias: Linear bias for the added variable.
         """
         return self.data.add_variable
 
-    def change_vartype(self, vartype, inplace=True):
+    def change_vartype(self, vartype: Optional[Vartype],
```

Comment:
`vartype` is not optional (not sure why the old docs claimed it was, probably a copy/paste error).

Reply:
Fixed